ccl/changefeedccl: TestAlterChangefeedAddTargetsDuringSchemaChangeError failed

ccl/changefeedccl.TestAlterChangefeedAddTargetsDuringSchemaChangeError [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20502104?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20502104?buildTab=artifacts#/) on master @ [004984b1a065cb1d4189dbbd4a2e5d36bc448158](https://github.com/cockroachdb/cockroach/commits/004984b1a065cb1d4189dbbd4a2e5d36bc448158):


```
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{10/0-20/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{10/0-20/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{20/0-30/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{20/0-30/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{30/0-40/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{30/0-40/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{40/0-50/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{40/0-50/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{50/0-60/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{50/0-60/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{60/0-70/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{60/0-70/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{70/0-80/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{70/0-80/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/1/9{80/0-90/0} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/1/9{80/0-90/0}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576877.367180048,1
    alter_changefeed_test.go:1439: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 0,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: false
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576879.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576882.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576885.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576888.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576891.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576894.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576897.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576900.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576903.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576906.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576909.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576912.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576915.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1405: span /Tenant/10/Table/106/{1-2} NONE 1758576918.000000000,0
    alter_changefeed_test.go:1431: handling span /Tenant/10/Table/106/{1-2}: true
    alter_changefeed_test.go:1451: condition failed to evaluate within 45s: from alter_changefeed_test.go:1458: waiting for checkpoint
--- FAIL: TestAlterChangefeedAddTargetsDuringSchemaChangeError/webhook (57.24s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/cdc
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestAlterChangefeedAddTargetsDuringSchemaChangeError.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/changefeedccl/alter_changefeed_test.go
+++ b/pkg/ccl/changefeedccl/alter_changefeed_test.go
@@ -1349,7 +1349,8 @@ func TestAlterChangefeedAddTargetsDuringSchemaChangeError(t *testing.T) {
 			return nil
 		}
 
-		testFeed := feed(t, f, `CREATE CHANGEFEED FOR foo WITH resolved = '1s', no_initial_scan`)
+		testFeed := feed(t, f, `CREATE CHANGEFEED FOR foo
+WITH resolved = '1s', no_initial_scan, min_checkpoint_frequency='1ns'`)
 		jobFeed := testFeed.(cdctest.EnterpriseTestFeed)
 		jobRegistry := s.Server.JobRegistry().(*jobs.Registry)
 

